ketcher-autotests/tests/test-data/KET/chem-connected-to-micro-structure-expected.ket

```diff
@@ -147,16 +132,14 @@
                     4,
                     5,
                     3,
-                    1,
-                    6
```

Comment:
Why it changed? Looks like it is not related to changes

Reply:
Reverted